privaterelay/settings.py

```diff
@@ -230,6 +230,8 @@ def _get_initial_middleware():
     },
 ]
 
+RELAY_FIREFOX_DOMAIN = config('RELAY_FIREFOX_DOMAIN', 'relay.firefox.com', cast=str)
+MOZMAIL_DOMAIN = config('MOZMAIL_DOMAIN', 'mozmail.com', cast=str)
```

Comment:
(I set `RELAY_FIREFOX_DOMAIN` to `unfck.email` and `MOZMAIL_DOMAIN` to `mozmail.unfck.email`)

The fact that these have default values seems to have messed up my aliases when I removed the env vars. My aliases were created with the domain `2` value, which became `MOZMAIL_DOMAIN`, and then all my aliases changed to `@mozmail.com` instead of going back to the `unfck.email` domain that they actually work on.

I think we're conflating env vars and DB values too much here. We may need to atone for the Relay Original Sin of NOT storing the domain values in the address records themselves.